ocp42

dacleyra opened the pull request:
Initial scripts for installing to Openshift 4.2 and CRC

stevenschader commented:
Can the SLEEP override logic be added to aid automated installs like:
# Over-ride sleep times if necessary for automation
SLEEP_LONG="${SLEEP_LONG:-5}"
SLEEP_SHORT="${SLEEP_SHORT:-1}"

dacleyra commented:
added sleep control var
added che CR which references specific kabaner-che image
ensure tekton-pipelines namespace exists before install
correct che subscription
subscribe to serverless-operator.v1.1.0 from kabanero catalog temporarily
